packages/backend/src/App.ts

```diff
@@ -102,6 +106,13 @@ export default class App {
                     this.lightdashConfig.rudder.dataPlaneUrl,
             },
         });
+        this.serviceRepository = new ServiceRepository({
```

Comment:
Eventually this will be done per-request, but this approach allows us to encapsulate App configuration already (via the `OperationContext` object + service overrides in `ServiceRepository`), while keeping the semantics consistent downstream for later.